frontend/src/metabase/admin/permissions/components/FilterableTree/FilterableTree.styled.tsx

```diff
@@ -1,5 +1,6 @@
 import styled from "@emotion/styled";
-import { color } from "metabase/lib/colors";
+import { Tree } from "metabase/components/tree";
```

Comment:
nit: should we change it to default exports as well?

Reply:
Good call, if you don't mind, I'd do that in a different PR